### PR TITLE
Feature/profile cards

### PR DESCRIPTION
**Pull request by vasyldzyuba**

### CODE REVIEW

File: src/app/component/user/profile/profile.component.html [hunk 1]
```diff
@@ -5,5 +5,15 @@
         vero delectus, ratione error tempore necessitatibus iure aperiam quam cupiditate facere minus perferendis quis.
         Fugiat, omnis!
     </div>
-    <div class="column right-column">Right</div>
+    <div class="column right-column">
+        <div class="right-cards">
+            <div [ngStyle]="{'backgroundColor': activity.backgroundColor}" class="card" *ngFor="let activity of cardsData;">
```
Comment (nvolodymyr) [inline src/app/component/user/profile/profile.component.html:10]: 10 line.
please, align in the column

File: src/app/component/user/profile/profile.component.scss [hunk 2]
```diff
@@ -33,6 +32,43 @@ section {
...
+          font-size: 15px;
+        }
+
+        .card-description {
+          font-size: 13px;
+          font-family: 'Open Sans';
+        }
+
+        .shape-img {
+
```
Comment (nvolodymyr) [inline src/app/component/user/profile/profile.component.scss:59]: delete gap

File: src/app/component/user/profile/profile.component.html (hunk 1 above)
Comment (vasyldzyuba) [reply src/app/component/user/profile/profile.component.html]: already fixed

File: src/app/component/user/profile/profile.component.scss (hunk 2 above)
Comment (vasyldzyuba) [reply src/app/component/user/profile/profile.component.scss]: fixed

File: src/app/component/user/profile/profile.component.html [hunk 3]
```diff
@@ -5,5 +5,17 @@
         vero delectus, ratione error tempore necessitatibus iure aperiam quam cupiditate facere minus perferendis quis.
         Fugiat, omnis!
```
Comment (nabermi2020) [inline src/app/component/user/profile/profile.component.html:6]: image
Please, add some fixes according to the mockup(font color, size, borders for items and bg for the side block)
Also create one more component and move appropriate logic and markup to this component.

Comment (vasyldzyuba) [reply src/app/component/user/profile/profile.component.html:6]: Got it. Already fixed.

**Pull request merged**